rewatch/src/build/build_types.rs

```diff
@@ -97,6 +97,7 @@ pub struct BuildState {
     pub module_names: AHashSet<String>,
     pub deleted_modules: AHashSet<String>,
     pub bsc_path: PathBuf,
+    pub runtime_path: Option<PathBuf>,
```

Comment:
Not 100% sold that this should be part of the `BuildState`.
Computing it in `compile::compile` is probably fine.
The fact that is is optional is a bit of a smell to make.
Clean doesn't use this.